proxy/http/HttpSM.cc

```diff
@@ -4059,6 +4061,16 @@ HttpSM::do_remap_request(bool run_inline)
     pending_action = remap_action_handle;
   }
 
+  // check if the overridden client cert filename is already attached to an existing ssl context
+  ats_scoped_str clientCert(Layout::relative_to(t_state.txn_conf->client_cert_filepath, t_state.txn_conf->client_cert_filename));
+  auto tCTX = params->getCTX(clientCert);
+
+  if (tCTX == nullptr) {
+    // make new client ctx and add it to the ctx list
+    auto tctx = ssl_NetProcessor.getNewCTX(clientCert);
+    params->InsertCTX(clientCert, tctx);
```

Comment:
Yes it would be cleaner. I reused the SSLInitClientContext logic to create the new CTX. That is why I kept it here. I will see if I can move the whole client context thing to SSLconfig